src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidUpdateResourcesTest.cs

```diff
@@ -458,12 +459,44 @@ protected override void OnClick()
 					.FirstOrDefault (x => x.Attribute("name").Value == "colorAccent");
 				Assert.IsNotNull (item, "The Style should contain an Item");
 				Assert.AreEqual ("@color/deep_purple_A200", item.Value, "item value should be @color/deep_purple_A200");
-				StringAssertEx.DoesNotContain ("AndroidResgen: Warning while updating Resource XML", b.LastBuildOutput,
+				Assert.IsFalse (StringAssertEx.ContainsText (b.LastBuildOutput, "AndroidResgen: Warning while updating Resource XML"),
```

Comment:
Does this test do a build with no changes? Wanting to make sure that this touch doesn't cause a targets to re-run: https://github.com/xamarin/xamarin-android/pull/2596/files#diff-db39bf07b89b50eaab8ab2001546310cR2378